backend/app.py

```diff
@@ -1,28 +1,38 @@
-from flask import Flask,request,redirect,jsonify
+from flask import Flask, request, redirect, jsonify
+from flask_sqlalchemy import SQLAlchemy
 import string
 import random
 import os
 
 
 app = Flask(__name__)
 
-@app.route('/hello', methods=['GET'])
-def hello_world():
-    return 'Hello, World!'
+# app configuration
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://root:mysecretpassword@127.0.0.1/my_database'
```

Comment:
We should not hard code credentials in code. We will learn to secure this later.